charts/k8tz/values.yaml

```diff
@@ -12,6 +12,15 @@ verbose: false
 webhook:
   failurePolicy: Fail
 
+  certManager:
+    enabled: false
+    secretTemplate: {}
+    duration: 5114d
```

Comment:
Error: INSTALLATION FAILED: admission webhook "webhook.cert-manager.io" denied the request: time: unknown unit "d" in duration "5114d"

Reply:
oh sorry, my bad. I was testing using helm template only